pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -272,6 +278,40 @@ var cnpIndexers = cache.Indexers{
 		}
 		return groupNames.List(), nil
 	},
+	HasServiceAccountsPeerIndex: func(obj interface{}) ([]string, error) {
+		cnp, ok := obj.(*secv1alpha1.ClusterNetworkPolicy)
+		if !ok {
+			return []string{}, nil
+		}
+		peerHasServiceAccounts := func(peers []secv1alpha1.NetworkPolicyPeer) bool {
+			for _, peer := range peers {
+				if peer.ServiceAccounts != nil {
+					return true
+				}
+			}
+			return false
+		}
+		if peerHasServiceAccounts(cnp.Spec.AppliedTo) {
+			return []string{strconv.FormatBool(true)}, nil
```

Comment:
using `strconv.FormatBool(true)` seems quite verbose when you could just use `"true"`?

Reply:
Thanks. Done.